portality/static/js/doaj.js

```diff
@@ -9,9 +9,11 @@ var doaj = {
         var openMenu = document.querySelector(".secondary-nav__menu-toggle");
         var nav = document.querySelector(".secondary-nav__menu");
 
-        openMenu.addEventListener('click', function() {
-            nav.classList.toggle("secondary-nav__menu-toggle--active");
-        }, false);
+        if (openMenu) {
+            openMenu.addEventListener('click', function () {
+                nav.classList.toggle("secondary-nav__menu-toggle--active");
+            }, false);
+        }
```

Comment:
This might actually be a general problem with the new admin layout, @sssoz you may want to pull this change into your UI dev branch